### PR TITLE
update: subject and content for invoice email dynamic

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Send Invoice Reminder #77

### CODE REVIEW

File: src/controllers/khbooks/invoice-controller.ts [hunk 1]
```diff
@@ -363,7 +363,7 @@ export const send = async (req: Request, res: Response) => {
 export const sendReminder = async (req: Request, res: Response) => {
   try {
     const { id } = req.params
-    await InvoiceService.sendInvoice(parseInt(id))
+    await InvoiceService.sendInvoice(parseInt(id), "Reminder")
```
Comment (jatnerubia) [inline src/controllers/khbooks/invoice-controller.ts:366]: Let's create an `SendInvoiceType` enum for this.

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a minor change ^^

File: src/controllers/khbooks/invoice-controller.ts (hunk 1 above)
Comment (chadiegilnerubia) [reply src/controllers/khbooks/invoice-controller.ts]: done :3

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**